addon.json

```diff
@@ -2,5 +2,5 @@
     "title": "Prop Hunters - Huskles Edition",
     "type": "gamemode",
     "tags": ["fun"],
-    "ignore": [".*", ".vscode", "*.md", "*.gma", "*.ps1"]
+    "ignore": [".*", ".vscode", "*.md", "*.gma", "*.ps1", "materials/husklesph/skull_license.txt"]

```

Comment:
Perhaps we just make a `licenses` folder somewhere (top level) and ignore it, rather than hardcoding the path.

Reply:
Just a note for the future: It was decided to keep the hardcoded path in here for the time being. If we get more licenses to keep track of then we'll move those to a top level folder and ignore that.